packages/gatsby-theme-store/src/sdk/search/useSearchFiltersFromPageContext.ts

```diff
@@ -1,17 +1,25 @@
 import { useLocation } from '@reach/router'
-import { useMemo } from 'react'
+import { useMemo, useContext } from 'react'
 
 import { format, parse } from './priceRange'
 import type { SearchPageContext } from '../../templates/search'
+import { Context } from '../region/Provider'
 
 export interface SelectedFacets {
   key: string
   value: string
 }
 
 const searchParamsFromURL = (pathname: string, params: URLSearchParams) => {
-  const smap = params.get('map')?.split(',')
-  const spath = pathname.split('/').slice(1)
+  const smap = params
+    .get('map')
+    ?.replace(/(,)?region$/g, '')
+    .split(',')
+
+  const spath = pathname
+    .replace(/(\/)?rId$/g, '')
```

Comment:
ah the `rId` string is used here it seems, is that right? it's ok although it seems kinda magical to me and at least some comments, on this file and on the other one, would be in order I think